examples/github-users/lib.ncl

```diff
@@ -57,7 +57,7 @@
             },
           }
       )
-    |> std.array.fold_left (&) {},
+    |> std.array.reduce_left (&),
```

Comment:
Is this array always non-empty? note that `reduce_left` will fail on an empty array, while `fold_left` doesn't.

Reply:
Oh, that's a really good point. I should just change this to `std.record.merge_all`, now that that's supposed to work again.